### PR TITLE
Remove ambiguity with precision

### LINKED ISSUE #3539: Ambiguity with Decimals
image

In some parts of the world, the numbers in the article above are read as a hundred thousand and five hundred thousand, which can be misleading. https://www.quora.com/Why-do-some-people-use-the-comma-instead-of-the-dot-for-larger-numbers

We need to refine the `human readable token`code to incorporate this https://github.com/w3f/polkadot-wiki/blob/master/scripts/fetch.mjs

### PR DESCRIPTION
**Pull request by DrW3RK**

### CODE REVIEW

**Comment by DrW3RK** [comment]
The rendering as of now
image

With this PR

image

While reviewing, check a few other pages on Polkadot Wiki and Kusama Guide to ensure nothing fuzzy is happening. Thanks!

File: components/RPC-Connection.jsx [hunk 1]
```diff
@@ -137,7 +137,13 @@ function applyFilter(value, filter, network, setReturnValue) {
 			value = parseFloat(value);
 			// TODO: this legacy function needs some attention (see https://github.com/w3f/polkadot-wiki/issues/3539)
```
Comment (alfarok) [inline components/RPC-Connection.jsx]:
```suggestion
```

File: components/RPC-Connection.jsx [hunk 2]
```diff
@@ -137,7 +137,13 @@ function applyFilter(value, filter, network, setReturnValue) {
...
 			// Apply precision
-    		value = `${(value / values[network].precision).toFixed(decimals)} ${values[network].symbol}`;
+			if (Number.isInteger(value / values[network].precision)) {
+				value = `${(value / values[network].precision)} ${values[network].symbol}`;
```
Comment (alfarok) [inline components/RPC-Connection.jsx]:
```suggestion
				value = `${value / values[network].precision} ${values[network].symbol}`;
```

**Review by alfarok — CHANGES_REQUESTED** [review]
Tested on several pages and works as expected:
image

Very minor changes suggested, nothing that impacts the rounding logic.

**Comment by DrW3RK** [comment]
Thanks! Please approve for merging

**Review by alfarok — APPROVED** [review]

**Pull request merged**